structural-imbalance/requirements.txt

```diff
@@ -1,3 +1,4 @@
 dwave-ocean-sdk==1.4.0
 matplotlib==2.2.4
+jsonschema==3.1.1
```

Comment:
Why exactly do we need this (if it has been removed as `dimod`'s dep)?

Reply:
The structural-imbalance unit tests uses `jsonschema` package to do some validations. Previously, it was just piggybacking off of dimod's installation of `jsonschema`, hence, the issue went unnoticed.